python/ql/src/Security/CWE-327/InsecureDefaultProtocol.qhelp

```diff
@@ -34,20 +34,24 @@
           <p>
             Both of the cases above should be updated to use a secure protocol
             instead, for instance by specifying
-            <code>ssl_version=PROTOCOL_TLSv1_1</code> as a keyword argument.
+            <code>ssl_version=PROTOCOL_TLSv1_2</code> as a keyword argument.
           </p>
           <p>
             Note that <code>ssl.wrap_socket</code> has been deprecated in
             Python 3.7. A preferred alternative is to use
             <code>ssl.SSLContext</code>, which is supported in Python 2.7.9 and
-            3.2 and later versions.
+            3.2 and later versions or the convenience function
+            <code>ssl.create_default_context</code>, which is supported in Python
+            3.4 and later versions.
           </p>
```

Comment:
Unfortunately, I can't add a suggestion here - GitHub complains about "deleted lines" which seems like a bug (in the tooltip if not in the functionality).

This is getting a little overlong and complex. I'd suggest breaking out into bullets for clarity and readability.

```
Note that <code>ssl.wrap_socket</code> has been deprecated in
Python 3.7. The recommended alternatives are:</p>
<ul>
  <li><code>ssl.SSLContext</code> - supported in Python 2.7.9,
         3.2, and later versions</li>
  <li><code>ssl.create_default_context</code> - a convenience function,
         is supported in Python 3.4 and later versions.</li>
</ul>
```

Reply:
Thanks, I hope we will be able to see the result of my attempt at adapting this :-)